### ISSUE TITLE
Proposal For Copywriting

### ISSUE THREAD

**Issue by ewuoso** [Dogfalo/materialize#5867]
Hello i went through your project and i want to say it is very nice. However i noticed you are missing out some important details like gaq, about us and privacy policy which users will be looking forward too. If you dont mind i can write that for you

contact me
email ewuosoolusesi@gmail.com

steemit.com/@ewuoso

**Comment by sigmatec** [comment]
I have to agree, I am really looking forward to reading the Privacy Policy. It's always the highlight of my day.

**Comment by ewuoso** [comment]
i need to ask some questions
